tests/ui/test_ui_tools.py

```diff
@@ -26,6 +26,24 @@
 SERVER_URL = "https://chat.zulip.zulip"
 
 
+@pytest.fixture(params=[
+        (key, button)
+        for keys, button in [
+            (keys_for_command('GO_UP'), 4),
+            (keys_for_command('GO_DOWN'), 5),
+        ]
+        for key in keys
+    ],
+    ids=lambda param: 'scroll_wheel_' + ('up' if param[1] == 4 else 'down')
+                      + '-key:{}-button:{}'.format(*param)
+)
+def mouse_press_key_button_pair(request):
```

Comment:
This is specifically for a mouse scrolling action, right? If so, maybe adjust the name accordingly.

Did you keep this out of `conftest.py` for a specific reason? (not that all fixtures have to be in there)

Reply:
I kept this here as it was only being used in `ui_tools.py`. Thanks for pointing this out; I have moved this to `conftest.py` as we could need it in other files as well.